src/test/unit_tests/test_s3_ops.js

```diff
@@ -438,69 +450,79 @@ mocha.describe('s3_ops', function() {
         });
         mocha.it('should list objects with text-file', async function() {
             // eslint-disable-next-line no-invalid-this
-            this.timeout(60000);
-            const res1 = await s3.listObjects({ Bucket: bucket_name }).promise();
+            this.timeout(6000000);
```

Comment:
Why this timeout so big? 
Unit tests should be small and fast, can we reduce it under 5 min?